plugins/inputs/tail/tail.go

```diff
@@ -41,10 +45,17 @@ type Tail struct {
 	offsets    map[string]int64
 	parserFunc parsers.ParserFunc
 	wg         sync.WaitGroup
-	ctx        context.Context
-	cancel     context.CancelFunc
-	acc        telegraf.TrackingAccumulator
-	sem        semaphore
+
+	acc telegraf.TrackingAccumulator
+
+	sync.Mutex
```

Comment:
This isn't necessary. Looks like it's here from a bad merge. (FYI You would probably have fewer problems using rebase instead of merge)